pages/linux/fcrackzip.md

```diff
@@ -0,0 +1,27 @@
+# fcrackzip
+
+> ZIP archive password cracking.
+
+- Bruteforce password with length 4 to 8 with only lowercase, uppercase and digit characters (order matters):
+
+`fcrackzip -b -l 4-8 -c aA1 {{archive}}`
+
+- Bruteforce password with length 3 with only lowercase characters, $ and % (verbose mode):
+
+`fcrackzip -v -b -l 3 -c a:$% {{archive}}`
+
+- Bruteforce password with length 4 with only lowercase and special characters:
+
+`fcrackzip -b -l 4 -c a! {{archive}}`
+
+- Bruteforce password with length 5 with only digits starting from password "12345":
```

Comment:
Perhaps this will read better?
```suggestion
- Brute-force a password containing only digits, starting from the password `12345`:
```

What do you mean by `starting from` here though?